src/main/java/edu/hm/hafner/analysis/parser/IssuesFilter.java

```diff
@@ -0,0 +1,188 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.regex.Pattern;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Issues;
+
+public class IssuesFilter {
+    private boolean includeFilterSet;
+    private boolean excludeFilterSet;
+    private Issues filteredIssues;
+    private Pattern patternIncludeFilename;
+    private Pattern patternIncludeCategory;
```

Comment:
Das sind viele Felder. Hast du in Betracht gezogen ,das Ganze in einer Collection oder eigenen Klasse zu speichern?